frontend/packages/topology/src/components/graph-view/TopologyControlBar.scss

```diff
@@ -0,0 +1,3 @@
+.odc-topology-control-bar{
```

Comment:
You can make it more obvious that you override a PatternFly variable here. Because `.pf-topology-control-bar` and `.odc-topology-control-bar` has the same CSS specific you can make even more clear with this:

```suggestion
.pf-topology-control-bar.odc-topology-control-bar {
  // Override absolute position of the control-bar
```

Reply:
Tried with that, but the CSS was not getting overridden.

Reply:
Okay fine with that. One nit, a space at the end would be nice :)

```suggestion
.odc-topology-control-bar {
```

Reply:
@jerolimov suggestion should work. I was curious about why it is not working, I tried it locally and it is working.
![image](https://user-images.githubusercontent.com/2561818/163789815-b00dd308-0ae3-4fe9-986f-43ddccdd7336.png)

And better to use `scss` format 
```suggestion
.pf-topology-control-bar {
  &.odc-topology-control-bar {
    left: var(--pf-global--spacer--sm);
  }
}
```

Reply:
Thanks @vikram-raj 
I have updated the code